packages/quality-gates/tests/test_exit_code_convention.py

```
@@ -0,0 +1,51 @@
+    "post-implementation-review.py",
+    "post-test-analysis.py",
+    "lint-on-save.py",
+    "test-tampering-detector.py",
+    "test-gate-checker.py",
+    "turn-end-summary.py",
+    "evaluation-set-checker.py",
+)
+
+_BLOCKING_HOOK = "post-test-analysis.py"
+
+# sys.exit(N) や os._exit(N) の N!=0 呼び出しを検出する（コメントアウト行は除外）。
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)